types/pagedjs-cli/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
```

Comment:
Add a `"type": "module"` to this `package.json` - the source package is esm-only.

Reply:
done ☺️